<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="lblDeposits">
         <property name="text">
          <string>Depósitos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtDeposit">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblBonus">
         <property name="text">
          <string>Bonus</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtBonus">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblProfits">
         <property name="text">
          <string>Beneficios</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtProfit">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblYield">
         <property name="text">
          <string>Yield</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtYield">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblWithdrawals">
         <property name="text">
          <string>Retiradas</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtTakeOut">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblTipsters">
         <property name="text">
          <string>Tipsters</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtTipster">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="lblBookies">
         <property name="text">
          <string>Casas</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblBank">
         <property name="text">
          <string>Banco</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Paypal</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Skrill</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblTotal">
         <property name="text">
          <string>Total</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtBookie">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtCc">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtPaypal">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtSkrill">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtTotal">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblBookiesLong">
     <property name="text">
      <string>En Casas de Apuestas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeBookie">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>300</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Casa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bank</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblMovements">
     <property name="text">
      <string>Movimientos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeMovement">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Casa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuenta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>€</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
